extension/script.js

```diff
@@ -394,10 +380,12 @@ function getNearestLink(element) {
 
 function processTitle(titleContainer) {
   if (titleContainer.hasListener) return;
-  Object.assign(titleContainer.style, {
-    display: 'flex',
-    gap: '15px',
-  });
+  titleContainer.classList.add("ta-title-container");
+
+  if (titleContainer.style.display === "-webkit-box") {
+    // Compatibility with DeArrow
+    titleContainer.style.setProperty("display", "flex", "important");
+  }
 
   titleContainer.classList.add('title-container');
```

Comment:
I don't think this class is used, but didn't want to remove it in case I was missing something. Figured I'd use a more specific class